.expeditor/scripts/verify/terraform_validate.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# TODO: sync this with terraform/versions.tf
+readonly terraform_version=0.12.13
```

Comment:
Not a blocking issue, however eventually it might be nice to consolidate reading the required TF version from the [versions.tf](https://github.com/habitat-sh/builder/blob/master/terraform/versions.tf) file.

Reply:
Addressed in https://github.com/habitat-sh/builder/pull/1252/commits/8cf42e72419021962f04da3c490f28da79634ac7